docs/2.guide/2.directory-structure/1.pages.md

```diff
@@ -356,8 +356,8 @@ function navigate(){
 </script>
 ```
 
-## Custom routing
+## Custom Routing
 
-As your app gets bigger and more complex, your routing might require more flexibility. For this reason, Nuxt directly exposes the router, routes and router options for customization in different ways.
+As your app gets bigger and more complex, your routing might require more flexibility. For this reason, Nuxt directly exposes the router, routes, and router options for customization in different ways.
```

Comment:
I believe the meaning here was options for both router and routes, hence no comma :)

Reply:
I believe both ways are fine, are they not? Plus, it would make more sense if the _router options_ was bundled with _router_ rather than routes, so I don't actually think it was meant to combine the 2..

Reply:
Had a read through the docs to try to understand what message this sentence is meant to convey, and what about:
```suggestion
As your app gets bigger and more complex, your routing might require more flexibility. For this reason, Nuxt exposes the router options, `pages:extend` hook, and utility functions for module authors, allowing for customization in different ways.
```

Also, semi-colons should be removed here and there's also a missing space:

![image](https://github.com/nuxt/nuxt/assets/48835293/dce6c8d9-9463-48e3-a4df-90469968fb6f)

Reply:
Done